README.md

```diff
@@ -47,6 +47,109 @@ The main design is to separate execution logic from execution. The execution can
 
 All you need to do is to add these annotations to the methods or bean classes you would like to achieve fault tolerance.
 
+## Using Microprofile Fault Tolerant Annotations
+
+### @Retry
+The `@Retry` annotation allows you to define a criteria on when to retry. The below example is configured to retry up to 5 times till the duration of 1000ms and when an Exception is occurred.
+
+```
+@GET
+    @Path("/attendee/retries")
+    @Produces(APPLICATION_JSON)
+    @Counted(name="io.microprofile.showcase.vote.api.SessionVote.getAllAttendees.monotonic.absolute",monotonic=true,absolute=true,tags="app=vote")
+    @Retry(maxRetries = 5, maxDuration= 1000, retryOn = {Exception.class})
+    public Collection<Attendee> getAllAttendeesRetries() {
+        Collection<Attendee> attendees = selectedAttendeeDAO.getAllAttendees();
+        if(attendees == null){
+            throw new RuntimeException("There must be attendees to run the meetings.");
+        }
+        return attendees;
+    }
+```
+
+### @Timeout
+The `@Timeout` annotation allows you to define a duration for timeout.It prevents from an execution to wait for ever. In the below example the timeout is 100ms after which the method will fail with a `TimeoutException`.
```

Comment:
- add space between sentences
- "annotation allows you to define a max amount of time that a method can be executed."

Reply:
the second bullet point is wrong. This annonation allows you to define for how long the timeout can occur after which it throws exception.